### DISCUSSION TITLE
"nohup helix" might be optimised?

### DISCUSSION THREAD

**Discussion by 707state** [helix-editor/helix#10288] [General]
I know terminal editors like vim and emacs shouldn't use nohup, but they redirect stdout to nohup.out and close with no problem. But Helix seems can't be redirected and closed properly. Might be my own problem coz I'm not familiar with Helix.

**Comment by pascalkuthe** [comment]
Helix just doesn't support non-interactive use. Probably not easy to add. I don't really see a usecase
